docs/DEVELOP_WITH_LOCAL_NODE.md

```diff
@@ -0,0 +1,334 @@
+# Develop with local node
+
+## Start local chain
+
+1. Pull this branch https://github.com/evmos/evmos/tree/release/v20.0.x
```

Comment:
nit: this is outdated since the team is pushing v50 as the v20 release will update you on which branch to use later.

Reply:
Ok thanks